### PR TITLE
CMS-98: Support returning info about created PR and optionally add comment to PR being closed.

### PR DESCRIPTION
**Pull request by kporras07**
### Overview
This pull request supports returning information about the created PR and adds option to add comment to the PR being closed.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes 
| Has tests?    | no
| BC breaks?    | no     
| Deprecations? | no 

### Description
- prCreate function now receives a new optional argument $response that will be filled with the response info after creating the PR.
- prClose function now receives a new optional argument $comment that will be posted to the PR just before closing it.

### CODE REVIEW

File: src/HubphAPI.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function prGet($org, $project, $id)
         return $this->gitHubAPI()->api('pull_request')->show($org, $project, $id);
     }
 
-    public function prCreate($org, $project, $title, $body, $base, $head)
+    public function prCreate($org, $project, $title, $body, $base, $head, &$response = null)
```
Comment (greg-1-anderson) [inline src/HubphAPI.php:76]: Let's make a new method that returns the response, and make `prCreate` call through to the new function, discard the result and return `$this` as it currently does.

Comment (greg-1-anderson) [reply src/HubphAPI.php:76]: Maybe name the new method `prOpen()`?

Comment (kporras07) [reply src/HubphAPI.php]: Done

File: src/HubphAPI.php [hunk 2]
```diff
@@ -73,15 +73,20 @@ public function prGet($org, $project, $id)
...
             'base' => $base,
             'head' => $head,
         ];
-        $response = $this->gitHubAPI()->api('pull_request')->create($org, $project, $params);
+        return $this->gitHubAPI()->api('pull_request')->create($org, $project, $params);
+    }
+
+    public function prCreate($org, $project, $title, $body, $base, $head)
+    {
+        $this->prOpen($org, $project, $title, $body, $base, $head);
```
Comment (greg-1-anderson) [inline src/HubphAPI.php:89]: `$response = `

Or move log up to prOpen

**Comment by greg-1-anderson** [comment]
Maybe run tests on GitHub actions while you're in here

File: src/HubphAPI.php (hunk 2 above)
Comment (kporras07) [reply src/HubphAPI.php]: whoops! Captured

**Comment by kporras07** [comment]
@greg-1-anderson added github actions tests

**Review by greg-1-anderson — APPROVED** [review]
Looks great; thanks for enabling tests.

**Pull request merged**